docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -127,6 +127,28 @@ _jboss.server.config.dir_.
 If your directory is located outside of _jboss.server.config.dir_, then
 you need to change the _path_ and _relative-to_ values appropriately.
 
+It is possible to encrypt the filesystem realm by configuring the credential-store and secret-key attributes for the realm.
+
+When creating a an encrypted filesystem realm, you will have to specify two more attributes.
```

Comment:
s/a an/an

Reply:
Would be good to reword this since only the `credential-store` is a required attribute. Perhaps something like:

To enable encryption for a `filesystem-realm`, there are a couple attributes that can be configured.